service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -207,6 +233,122 @@ void shouldNotifyPartiesOnOrderSubmission() throws IOException, NotificationClie
         }
     }
 
+    @Nested
+    class CaseManagementOrderNotificationTests {
+        private final Map<String, Object> cmoOrderIssuedNotificationParameters = getCMOIssuedNotificationParameters();
+        private final Map<String, Object> expectedCMOOrderIssuedNotificationParametersForRepresentative =
+            getExpectedCMOOrderIssuedNotificationParametersForRepresentative();
+
+        private NotificationHandler cmoNotificationHandler;
+
+        @BeforeEach
+        void setup() {
+            // did this to enable ObjectMapper injection
+            cmoNotificationHandler = new NotificationHandler(hmctsCourtLookupConfiguration, cafcassLookupConfiguration,
```

Comment:
Not the most important but would make this cleaner, you could make beans in the `LookupTestConfig` to mock the lookup configuration classes. I believe there is already a `HmctsCourtLookupConfiguration` example already in there